fuels-signers/src/wallet.rs

```diff
@@ -91,11 +88,8 @@ impl Wallet {
         private_key: SecretKey,
         provider: Provider,
     ) -> Result<Self, WalletError> {
-        let secp = Secp256k1::new();
-
-        let public = PublicKey::from_secret_key(&secp, &private_key).serialize_uncompressed();
-        let public = Bytes64::try_from(&public[1..])?;
-        let hashed = Hasher::hash(public);
+        let public = PublicKey::from(&private_key);
+        let hashed = public.hash();
```

Comment:
same remark as below regarding using `Result` -> it's not useful if you can't get "an error" out

Reply:
(I think at least lol)